File: .tmp_py4013/batch_04_sources/2527_find_xor_beauty_of_array.js
// LeetCode 2527 - Find Xor-Beauty of Array
// https://leetcode.com/problems/find-xor-beauty-of-array/

/**
 * @param {number[]} nums
 * @return {number}
 */
var xorBeauty = function(nums) {
    let ans = 0;
    for (const x of nums) ans ^= x;
    return ans;
};

File: .tmp_rs4013/rust_check09/2527_find_xor_beauty_of_array.rs
struct Solution;

// LeetCode 2527 - Find Xor-Beauty of Array
// https://leetcode.com/problems/find-xor-beauty-of-array/

impl Solution {
    pub fn xor_beauty(nums: Vec<i32>) -> i32 {
        let mut ans = 0;
        for x in nums {
            ans ^= x;
        }
        ans
    }
}

fn main() {}
